bower.json

```diff
@@ -41,7 +41,8 @@
     "angular-extension-registry": "1.2.6",
     "ng-sortable": "1.3.4",
     "ui-select": "angular-ui-select#0.16.1",
-    "matchHeight": "0.6.0"
+    "matchHeight": "0.6.0",
+    "angular-key-value-editor": "https://github.com/benjaminapetersen/angular-key-value-editor.git#v0.0.9"
```

Comment:
Just making a note on our conversation about pulling `angular-key-value-editor` under the `openshift` org temporarily, then possibly to `patternfly` as a better long term resting place.